src/test/java/uk/gov/hmcts/reform/roleassignment/domain/service/drools/StaffCategoryOrgRoleTest.java

```diff
@@ -88,13 +98,15 @@ void shouldRejectOrgRequestedRoleForTCW_WrongClientID_S003() {
     @Test
     void shouldRejectOrgValidationForTCW_WrongRoleCategory_S004() {
         //clientId check not implemented yet
-        //assignmentRequest.getRequest().setClientId("am_org_role_mapping_service");
+        assignmentRequest.getRequest().setClientId("am_org_role_mapping_service");
         assignmentRequest.setRequestedRoles(getRequestedOrgRole());
         assignmentRequest.getRequestedRoles().stream().forEach(roleAssignment -> {
             roleAssignment.setRoleCategory(RoleCategory.JUDICIAL);
             roleAssignment.setRoleType(RoleType.ORGANISATION);
             roleAssignment.setRoleName("tribunal-caseworker");
             roleAssignment.setGrantType(STANDARD);
+            roleAssignment.setStatus(CREATE_REQUESTED);
+            roleAssignment.setRoleType(CASE);
```

Comment:
why you are setting roletype= case in staff org test file? Please remove it.

Reply:
Done